### PR TITLE
Fix Yarn test

### PR DESCRIPTION
**Pull request by JulienMrgrd**
This fix add the new RouteService (see #99) to the Health component tests.

### CODE REVIEW

File: src/test/javascript/spec/app/admin/health/health.component.spec.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { ComponentFixture, TestBed, async } from '@angular/core/testing';
-import { MockBackend } from '@angular/http/testing';
-import { Http, BaseRequestOptions } from '@angular/http';
-import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
-import { JhiHealthCheckComponent } from '../../../../../../main/webapp/app/admin/health/health.component';
-import { JhiHealthService } from '../../../../../../main/webapp/app/admin/health/health.service';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
```
Comment (deepu105) [inline src/test/javascript/spec/app/admin/health/health.component.spec.ts]: should have space after curly brace

Comment (JulienMrgrd) [inline src/test/javascript/spec/app/admin/health/health.component.spec.ts]: Ahhh, I don't know why my Intellij formatter remove spaces sometimes... Now it's ok in all TypeScript files :)

Comment (JulienMrgrd) [inline src/test/javascript/spec/app/admin/health/health.component.spec.ts]: Yes, indeed. I've changed my Intellij TypeScript formatter to respect as possible the actual code format.
Now, I hope it's OK :)
I've just reformat all *.ts and *.css in a new commit

Comment (deepu105) [inline src/test/javascript/spec/app/admin/health/health.component.spec.ts]: This file still seems to have wrong formatting

Comment (JulienMrgrd) [inline src/test/javascript/spec/app/admin/health/health.component.spec.ts]: Rahhh, I've applied the formatting on the webapp package but I've forgot the test/javascript/spec package .... Sorry for that. I will include this change in my next PR.

**Pull request merged**